www/src/pages/themes/Product.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link, Route, Switch } from 'react-router-dom';
+import { Grid, Menu, Button, Card, Image } from 'semantic-ui-react'
+import Jumbotron from '../../components/Jumbotron'
+import Details from './Details'
+import Reviews from './Reviews'
+import daniel from './daniel.jpg'
+
+class Product extends React.Component {
+  state = { activeItem: 'home' }
```

Comment:
Do we need constructors any more? Not sure what is the difference some times.
Do these act as default instance variables where you do not have to call `this.state`?